src/server/__tests__/schema.test.js

```diff
@@ -45,13 +45,14 @@ describe('Schema', () => {
     }`;
   test('could create query schemas', async () => {
     const querySchema = getQuerySchema(config.esConfig);
-    expect(removeSpacesNewlinesAndDescriptions(querySchema))
+    expect(removeSpacesNewlinesAndDes(querySchema))
       .toEqual(removeSpacesAndNewlines(expectedQuerySchemas));
   });
 
   const expectedTypesSchemas = `
     type Subject {
       gen3_resource_path: String,
+      visits(days_to_visit:[Int],visit_label:[String]):[visits],
```

Comment:
is all the nested properties going to be available as an argument for filtering? for example user could filter subjects with visits by `visiti_label` values?